pkg/virt-handler/non-root.go

```diff
@@ -113,11 +114,12 @@ func getTapDevices(vmi *v1.VirtualMachineInstance) []string {
 		}
 	}
 
+	networkNameScheme := namescheme.CreateNetworkNameScheme(vmi.Spec.Networks)
 	tapDevices := []string{}
 	for _, net := range vmi.Spec.Networks {
-		_, ok := macvtap[net.Name]
-		if ok {
-			tapDevices = append(tapDevices, net.Multus.NetworkName)
+		_, isMacvtapNetwork := macvtap[net.Name]
```

Comment:
Previously the nad name was used for the macvtap and now we are using the hashed interface/network name? Where is the code changing the name of the macvtap?

Reply:
I don't follow the question.

In this PR we are changing the interface name requested from multus for **all** types of interfaces.

Reply:
We clarified offline that apparently this code had a bug since it used the NAD name to calculate the tap name. The e2e tests passed just because the NAD was called `net1`. Please move this bug fix to a separate commit with a proper description.

Reply:
I would also ask for e2e test to exercise it if I may

Reply:
@xpivarc , the test is there.

> The e2e tests passed just because the NAD was called net1

Now that the names are not the same as the NAD, this should be fine.

Can you confirm @maiqueb ?

Reply:
Yes.